apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex

```diff
@@ -165,6 +81,29 @@
               <%= BlockScoutWeb.Cldr.Number.to_string!(@address_count, format: "#,###") %>
             </span>
           </div>
+
+            <%= if Map.has_key?(@chart_config, :market) do %>
+              <%= if Enum.member?(@chart_config.market, :price) do %>
+                <div class="dashboard-banner-network-stats-item dashboard-banner-network-stats-item-5">
+                    <h2 class="dashboard-banner-network-stats-label">
+                        <%= gettext "Price" %>
+                    </h2>
+                    <span class="dashboard-banner-network-stats-value" data-selector="exchange-rate" data-wei-value="<%= Explorer.Chain.Wei.from(Decimal.new(1), :ether).value %>" data-usd-exchange-rate="<%= @exchange_rate.usd_value %>">
+
+                        </span>
+                </div>
+              <% end %>
+              <%= if Enum.member?(@chart_config.market, :market_cap) do %>
+                <div class="dashboard-banner-network-stats-item dashboard-banner-network-stats-item-6">
+                    <h2 class="dashboard-banner-network-stats-label">
+                        <%= gettext "Market Cap" %>
+                    </h2>
+                    <% total_market_cap = market_cap(@market_cap_calculation, @exchange_rate) %>
+                    <span class="dashboard-banner-network-stats-value" data-selector="market-cap" data-usd-value="<%= total_market_cap %>">
+                        </span>
+                </div>
+              <% end %>
+          <% end %>
```

Comment:
I think we need to format this part, using indentation with 2 spaces

Reply:
Np~, will update IDE.